test/syscalls/linux/msgqueue.cc

```diff
@@ -26,9 +26,14 @@ namespace gvisor {
 namespace testing {
 namespace {
 
-constexpr int msgMax = 8192;   // Max size for message in bytes.
+constexpr int msgMnb = 16384;  // Maximum number of bytes in a queue.
```

Comment:
Maybe indicate where in linux source these constants are coming from.

Reply:
Done